data/src/main/java/io/gnosis/data/backend/GatewayApi.kt

```diff
@@ -62,14 +62,14 @@ interface GatewayApi {
     suspend fun loadTransactionsHistory(
         @Path("chainId") chainId: BigInteger,
         @Path("address") address: String,
-        @Query("timezone_offset") timezoneOffset: Int = TimeZone.getDefault().getOffset(Date().time)
+        @Query("timezone_offset") timezoneOffset: Int = 0
     ): Page<TxListEntry>
 
     @GET("/v1/chains/{chainId}/safes/{address}/transactions/queued")
     suspend fun loadTransactionsQueue(
         @Path("chainId") chainId: BigInteger,
         @Path("address") address: String,
-        @Query("timezone_offset") timezoneOffset: Int = TimeZone.getDefault().getOffset(Date().time)
+        @Query("timezone_offset") timezoneOffset: Int = 0
```

Comment:
Do we need this parameter at all then ?

Reply:
I wanted to leave it because it is in the api. And maybe someone wants to use it in the future. I just set the default to 0.

Reply:
Got referenced here from #3327.  This change should bring inaccurate grouping of transactions by the date. 

Since the client doesn't report the offset to the server, the transaction list can have some transactions appearing in the wrong day, if those transactions made closer to midnight.